src/validation/manual.address.validation.ts

```diff
@@ -0,0 +1,220 @@
+import { Address } from "@companieshouse/api-sdk-node/dist/services/officer-filing";
+import { ManualAddressValidationType, ValidationError } from "../model/validation.model";
+import { COUNTRY_LIST, UK_COUNTRY_LIST } from "../utils/properties";
+
+const REGEX_FOR_MANUAL_ADDRESS = "^[-,.:; 0-9A-Z&@$£¥€'\"«»?!/\\\\()\\[\\]{}<>*=#%+ÀÁÂÃÄÅĀĂĄÆǼÇĆĈĊČÞĎÐÈÉÊËĒĔĖĘĚĜĞĠĢĤĦÌÍÎÏĨĪĬĮİĴĶĹĻĽĿŁÑŃŅŇŊÒÓÔÕÖØŌŎŐǾŒŔŖŘŚŜŞŠŢŤŦÙÚÛÜŨŪŬŮŰŲŴẀẂẄỲÝŶŸŹŻŽa-zſƒǺàáâãäåāăąæǽçćĉċčþďðèéêëēĕėęěĝģğġĥħìíîïĩīĭįĵķĺļľŀłñńņňŋòóôõöøōŏőǿœŕŗřśŝşšţťŧùúûüũūŭůűųŵẁẃẅỳýŷÿźżž]*$";
```

Comment:
Should these regex's not be neater from another validation util file to make them reusable in the case of testing?

Reply:
Yes, I agree to this. There have been some inconsistencies across some ACs covering character validation. I will compare this with occupation and move it to util class.